### PR TITLE
add dev-repo to ocsfml

### PR DESCRIPTION
**Pull request by zapashcanon**
@kyoDralliam @JoeDralliam could you upstream the fix ?

Alsot, the url.src looks really weird to me, it points to a fork which is not the main repository and points to the `master` tarball. Is wonder if this it going to break as soon as someone push to that branch ? What do you think @kit-ty-kate ?

### CODE REVIEW

File: packages/ocsfml/ocsfml.2.0/opam [hunk 1]
```diff
@@ -15,7 +15,8 @@ synopsis: "Binding to the C++ SFML gaming library."
 description: """
 The C++ library SFML (2.0) must be installed before any installation of this binding.
 See http://www.sfml-dev.org/index.php for more information."""
+dev-repo: "git+https://github.com/JoeDralliam/Ocsfml.git"
 url {
-  src: "https://github.com/KyoDralliam/Ocsfml/tarball/master"
+  src: "https://github.com/KyoDralliam/Ocsfml/archive/5a90ae9596861166bf69a36423318924595cd172.tar.gz"
```
Comment (mseri) [inline packages/ocsfml/ocsfml.2.0/opam:20]: 2021-09-14 11:36.52: Error in ocsfml.2.0: Failed to download the archive. Details: https://github.com/KyoDralliam/Ocsfml/archive/5a90ae9596861166bf69a36423318924595cd172.tar.gz (Bad checksum, expected md5=bf823023c9dcc9de9b41ec8eb77861cb)

Comment (zapashcanon) [reply packages/ocsfml/ocsfml.2.0/opam:20]: I checked with pkgdiff and the content is unchanged, I'll update the checksum.

Comment (mseri) [reply packages/ocsfml/ocsfml.2.0/opam:20]: Thanks!

**Comment by mseri** [comment]
Looks like all the machines in the CI have the wrong version of the library :P

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**